locales/en/messages.json

```diff
@@ -3,6 +3,7 @@
     "success": {
       "heading": "Move requested",
       "heading_proposed": "Move sent for review",
+      "heading_youth_transfer": "Move sent for review",
```

Comment:
Does this addition to the locales need to be part of this PR?

Reply:
That's true. I toyed with splitting this into separate PRs but making the second change means no need to add the move transformer in the end so grouped them together.

I'm happy to split them up if it's easier but the second will depend on the first?